presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/AbstractResourceConfigurationManager.java

```diff
@@ -13,6 +13,7 @@
  */
 package com.facebook.presto.resourceGroups;
 
+import com.facebook.presto.resourceGroups.reloading.ManagerSpec;
```

Comment:
why has `ManagerSpec` been moved to the `reloading` package ?

Reply:
my bad, this was an accident